lib/urbanopt/geojson/building.rb

```diff
@@ -185,17 +141,16 @@ def create_other_buildings(surrounding_buildings, model, origin_lat_lon, runner,
             floor_print.each do |point|
               building_points << point
             end
-            # subsequent polygons are holes, we do not support them
```

Comment:
Was this deleted because it is no longer true? If it is still true, this seems like a very valuable comment to include in the code.